lstchain/reco/utils.py

```diff
@@ -277,16 +278,17 @@ def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az):
     y = np.array([1,1]) * u.m
 
     """
-    pointing_direction = HorizonFrame(alt=pointing_alt, az=pointing_az)
+    pointing_direction = AltAz(alt=pointing_alt, az=pointing_az)
 
-    source_pos_in_camera = NominalFrame(array_direction=pointing_direction,
-                                        pointing_direction=pointing_direction,
-                                        x=pos_x/focal * u.rad,
-                                        y=pos_y/focal * u.rad,
-                                        )
+    camera_frame = CameraFrame(focal_length=focal,
+                               telescope_pointing=pointing_direction
+    )
 
-    return source_pos_in_camera.transform_to(pointing_direction)
+    camera_coord = SkyCoord(pos_x, pos_y, frame=camera_frame)
 
+    horizon = camera_coord.transform_to(AltAz())
```

Comment:
here you can also use the `horizon_frame` defined above without  using a new AltAz()